demo/index.html

```diff
@@ -174,6 +184,18 @@ <h1>${this.title}</h1>
       href: '#',
     }];
 
+    function extraActionClickHandler(e) {
+      const audio = menus.find(m => m.title === 'Audio Menu');
+      showPrimary = !showPrimary;
+
+      audio.actionButton = showPrimary ? primaryActionButton : secondaryActionButton;
```

Comment:
@dualcnhq this demo shows the toggling icons persist with this branch.

current version is 0.2.0-alpha3 to test updates with menu close bug we found today.
i've added it to www-isa to verify. in your branch, please do update bookreader's use to 0.2.0-alpha3